### ISSUE TITLE
long delay after biometrics button on the unlock screen was pressed

### ISSUE THREAD

**Issue by WalterZwei** [beemdevelopment/Aegis#1194]
### Version

2.2.2

### Source

Google Play

### Vault encryption

Yes (with biometric unlock)

### Device

BQ Aquaris X Pro 

### Android version

LineageOS 20 ( = Android 13), with Google Apps/Services

### ROM

LineageOS 20 ( = Android 13), with Google Apps/Services

### Steps to reproduce

1. Open App
2. On the login screen the biometric button appears
3. I press the fingerprint sensor
4. The button changes immediately to a green checkmark and vanish after ca. 0.3 seconds
5. --> for about 3.5 seconds, only the lockscreen is visible (the screen with username and password), nothing else happens <---
6. after those 3.5 seconds without further interactions, the lockscreen vanished and the code screen appears

### What do you expect to happen?

the code screen has to appear without delay after step 4

Remark: On a Nokia 8.3 (Android 12) the delay is shorter, approx. 0.5 seconds

### What happens instead?

a 3.5 second delay, see step 5

**Comment by alexbakker** [comment]
Could you [capture a log using ADB](https://github.com/beemdevelopment/Aegis/blob/master/CONTRIBUTING.md#capturing-a-log-with-adb) of you reproducing the issue and share it here?

Sounds like this may be similar to: #1183. If it is, I'm afraid there isn't much we can do on our end to improve performance here. We can show a "loading" dialog instead of freezing the UI, but you'll probably still be stuck waiting for a few seconds.

**Comment by WalterZwei** [comment]
Thanks for the quick answer. 

I'll try to log it, but it may take me a while to upload this

It see it as a minor bug, only a little annoyance on my spare phone from 2017, so please do not spend to much time on this issue.

**Comment by alexbakker** [comment]
Closing as a duplicate of #1183 since this sounds very similar and we haven't gotten any new information.
